e2e/preference/create.spec.ts

```diff
@@ -0,0 +1,58 @@
+import type { PreferenceCreateData } from '@src/clients/preference/create/types';
+import MercadoPago, { Preference } from '@src/index';
+import { config } from '../e2e.config';
+
+describe('Preference IT, create', () => {
+	test('should create a request with success', async () => {
```

Comment:
Regarding the test description, I think this `request` got a little lost, is better something like `create Preference and match response object`